packages/fluentui/docs/src/examples/components/Datepicker/Types/DatepickerExample.shorthand.tsx

```diff
@@ -1,6 +1,6 @@
-import { Datepicker } from '@fluentui/react-northstar';
+import { Datepicker, DayOfWeek } from '@fluentui/react-northstar';
 import * as React from 'react';
 
-const DatepickerExample = () => <Datepicker />;
+const DatepickerExample = () => <Datepicker firstDayOfWeek={DayOfWeek.Tuesday} />;
```

Comment:
Do we want to change the basic example? We already have a special firstDayOfWeek example.

Reply:
No, that's left from my debugging, good catch :)